code/stable-website/index.html

```diff
@@ -32,7 +32,7 @@ <h1></h1>
         <div id="Figure:Drasil">
           <figure>
             <img src="./images/Icon.png" alt="" width="50.0%" >
-            <figcaption></figcaption>
+            <figcaption>Figure: </figcaption>
```

Comment:
Changing the `image` function will also affect `drasil-website`. Maybe we should add a caption to drasil's icon to ensure that every image has a caption?

Reply:
That particular image is for the header of the website, which, in my opinion, wouldn't make sense to have a caption for.

![image](https://github.com/user-attachments/assets/eed0a082-18ae-4014-b985-e6fae4e22225)

Reply:
We can either include a caption for every image, as you suggested, or modify the image function to append `"Figure:"` only if the caption is non-empty. If we choose the first option, we enforce that *all* images *must* have a caption. In this case, we should handle empty captions by generating an error, as the issue of `"Figure:"` appearing with an empty caption would persist since there is nothing stopping a user from defining an empty caption.

That being said, others may have differing opinions, so we should wait for their input.

Reply:
Great observation and solution @BilalM04.  I agree that there are cases where we don't want a caption and in those cases the term `Figure:` should not appear.

Reply:
The solution there would be to use a `Maybe` so that `Nothing` would indicate an empty caption -- that way it is clear that this is *intentional*. An empty string caption might well be an accident.